metricbeat/module/elasticsearch/node_stats/data.go

```diff
@@ -140,6 +141,10 @@ func eventsMapping(r mb.ReporterV2, content []byte) error {
 			errs = append(errs, event.Error)
 		}
 
+		name, _ := event.MetricSetFields.GetValue("name")
+		event.ModuleFields.Put("node.name", name.(string))
```

Comment:
We should probably check the type conversion here? Also what happens if above returns and error and `name` does not exist? Will `name` be nil?

Reply:
I see. Agree that it should not break here but what if someone modifies the other part of the code and does not thing about this part? I normally prefer to be on the safe side.